mtp_noms_ops/apps/security/tasks.py

```diff
@@ -24,26 +25,25 @@ def zip_credit_export(attachment_name, output):
 
 
 @spoolable(body_params=('user', 'session', 'filters'))
-def email_credit_csv(*, user, session, endpoint_path, filters, export_description,
-                     attachment_name, zip_attachment=True):
+def email_credit_xlsx(*, user, session, endpoint_path, filters, export_description,
+                      attachment_name, zip_attachment=True):
```

Comment:
Yeah, I was thinking about that, but it does make some small difference (like 3%), and for really large files that might be a significant number of bytes. Probably only relevant if someone does actually try and download the entire database though...